ERROR - 2024-07-20 09:20:03 --> mysqli_sql_exception: Unknown column 'cheque_note' in 'field list' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ch...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ch...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ch...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ch...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\aitsun-erp\app\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
#11 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-07-20 09:20:03 --> Unknown column 'cheque_note' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cheques` (`company_id`, `cheque_no`, `cheque_date`, `cheque_department`, `cheque_title`, `amount`, `status`, `cheque_category`, `cheque_note`, `created_date`, `added_by`) VALUES (:company_id:, :cheque_no:, :cheque_date:, :cheque_department:, :cheque_title:, :amount:, :status:, :cheque_category:, :cheque_note:, :created_date:, :added_by:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-20 09:21:34 --> mysqli_sql_exception: Unknown column 'cheque_note' in 'field list' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ch...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ch...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ch...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ch...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\aitsun-erp\app\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
#11 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-07-20 09:21:34 --> Unknown column 'cheque_note' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cheques` (`company_id`, `cheque_no`, `cheque_date`, `cheque_department`, `cheque_title`, `amount`, `status`, `cheque_category`, `cheque_note`, `created_date`, `added_by`) VALUES (:company_id:, :cheque_no:, :cheque_date:, :cheque_department:, :cheque_title:, :amount:, :status:, :cheque_category:, :cheque_note:, :created_date:, :added_by:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-20 09:56:15 --> mysqli_sql_exception: Unknown column 'cheque_note' in 'field list' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ch...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ch...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ch...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ch...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\aitsun-erp\app\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
#11 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-07-20 09:56:15 --> Unknown column 'cheque_note' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cheques` (`company_id`, `cheque_no`, `cheque_date`, `cheque_department`, `cheque_title`, `amount`, `status`, `cheque_category`, `cheque_note`, `created_date`, `added_by`) VALUES (:company_id:, :cheque_no:, :cheque_date:, :cheque_department:, :cheque_title:, :amount:, :status:, :cheque_category:, :cheque_note:, :created_date:, :added_by:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-20 09:57:51 --> mysqli_sql_exception: Unknown column 'cheque_note' in 'field list' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ch...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ch...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ch...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ch...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\aitsun-erp\app\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
#11 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-07-20 09:57:51 --> Unknown column 'cheque_note' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cheques` (`company_id`, `cheque_no`, `cheque_date`, `cheque_department`, `cheque_title`, `amount`, `status`, `cheque_category`, `cheque_note`, `created_date`, `added_by`) VALUES (:company_id:, :cheque_no:, :cheque_date:, :cheque_department:, :cheque_title:, :amount:, :status:, :cheque_category:, :cheque_note:, :created_date:, :added_by:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Cheque_management.php(221): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-20 09:59:31 --> mysqli_sql_exception: Unknown column 'cheque_note' in 'field list' in C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\aitsun-erp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ch...', 0)
#1 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ch...')
#2 C:\xampp\htdocs\aitsun-erp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ch...')
#3 C:\xampp\htdocs\aitsun-erp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ch...', Array, false)
#4 C:\xampp\htdocs\aitsun-erp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\aitsun-erp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\aitsun-erp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\aitsun-erp\app\Controllers\Cheque_management.php(220): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
#10 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
#11 C:\xampp\htdocs\aitsun-erp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\aitsun-erp\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-07-20 09:59:31 --> Unknown column 'cheque_note' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cheques` (`company_id`, `cheque_no`, `cheque_date`, `cheque_department`, `cheque_title`, `amount`, `status`, `cheque_category`, `cheque_note`) VALUES (:company_id:, :cheque_no:, :cheque_date:, :cheque_department:, :cheque_title:, :amount:, :status:, :cheque_category:, :cheque_note:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Cheque_management.php(220): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cheque_management->add_cheque()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cheque_management))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
